examples/prisma-example/apps/api/src/resources/user/actions/remove.ts

```diff
@@ -0,0 +1,34 @@
+import { userService } from 'resources/user';
+
+import { AppKoaContext, AppRouter, Next } from 'types';
+
+type ValidatedData = never;
+type Request = {
+  params: {
+    id: string;
+  };
+};
+
+async function validator(ctx: AppKoaContext<ValidatedData, Request>, next: Next) {
+  const isUserExists = Boolean(
```

Comment:
```suggestion
  const isUserExists = Boolean(
```

use `count()` for all `exists` vars